### PR TITLE
Update UG Diagrams and Manual testing

### PR DESCRIPTION
**Pull request by markusyeo**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -450,6 +450,8 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 ### Glossary
 
 * **Mainstream OS**: Windows, Linux, Unix, OS-X
+* **Person**: A person who has gotten Dengue
```
Comment (sumhungyee) [inline docs/DeveloperGuide.md:453]:
```suggestion
* **Person**: A person who has contracted Dengue fever.
```

**Review by sumhungyee — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ The rest of the App consists of four components.
 
 * [**`UI`**](#ui-component): The UI of the App.
 * [**`Logic`**](#logic-component): The command executor.
-* [**`Model`**](#model-component): Holds the data of the App in memory.
+* [**`Model`**](#model-component): Holds tqgiothe data of the App in memory.
```
[the remark this answers is not in the document]
Comment (axmszr) [reply docs/DeveloperGuide.md]: LOL

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -450,6 +450,8 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 ### Glossary
 
 * **Mainstream OS**: Windows, Linux, Unix, OS-X
+* **Person**: A person who has contracted Dengue fever.
+* **CSV file: Comma-separated values file
```
Comment (axmszr) [inline docs/DeveloperGuide.md:454]:
```suggestion
* **CSV file**: Comma-separated values file
```

File: docs/images/BetterModelClassDiagram.png
Comment (axmszr) [inline docs/images/BetterModelClassDiagram.png:1]: Maybe we should remove the diamonds? Better not risk incorrect composition claims.

**Review by axmszr — CHANGES_REQUESTED** [review]
Some nits, but thanks for the work!

**Review by axmszr — APPROVED** [review]

**Comment by axmszr** [comment]
We may need to touch up on the DG later!

**Pull request merged**